docs/X64WINE.md

```diff
@@ -12,6 +12,7 @@ The general procedure for installing Wine64 & Wine for Box64 & Box86 is to...
  - Unzip or dpkg the install files into one folder
  - Move that folder to the directory that you wish Wine to run from (often `~/wine/` by default)
  - Go to `/usr/local/bin` and make symlinks or scripts that will point to your main wine binaries.
+ - Ensure that shared libraries are present in the search paths `usr/lib/x86_64-linux-gnu/` (for x86_64) or `usr/lib/i386-linux-gnu/` (for i386) if you only have the box64 binary.
```

Comment:
We use `/usr/lib/box64-x86_64-linux-gnu/` and `/usr/lib/box64-i386-linux-gnu/` paths now, the old paths are still supported, but deprecated.